apps/src/templates/BaseDialog.jsx

```diff
@@ -85,13 +87,13 @@ var BaseDialog = React.createClass({
       modalBodyClassNames = "";
       modalBodyStyle = {
         background: `#fff top center url(${this.props.assetUrl('media/dialog/achievement_background.png')}) no-repeat`,
-        height: 480,
+        height: this.props.fixedHeight,
```

Comment:
We intentionally aren't doing that, so that when a height is not provided the dialog will flow to the size of its content.  I think this is the best default because it's most likely to 'just work' for future consumers of the component if they don't bother to set this prop.